1.1/spec/change-log.md

```diff
@@ -106,3 +114,31 @@ The E048 error description in `validation-codes.md` is corrected to remove menti
 #### Fix E070 description
 
 The E070 error description in `validation-codes.md` is corrected to refer to `extension` rather than `key` (which was left from an earlier draft). (Issue [#573](https://github.com/OCFL/spec/issues/573))
+
+## Changes from OCFL v1.1.0 to v1.1.1
+
+[Version 1.1.1 of the OCFL Specification](https://ocfl.io/1.1/spec/) is a [patch version](https://semver.org/) update to the [OCFL Specification v1.1.0](https://ocfl.io/1.1.0/spec/). There are only clarifications.
+
+### Clarifications in v1.1.1
+
+#### Reword filesystem case sensitivity comments
+
+Version 1.1.0 had an unenforceable MUST regarding filesystem case preservation. The [Filesystem Features](https://ocfl.io/1.1/spec/#filesystem-features) section was change to instead point out that implementation over filesystems that either do not preserve case or are not case sensitive require great care, including making appropriate choices for file paths and filenames. (Issue [#528](https://github.com/OCFL/spec/issues/528))
```

Comment:
change->changed

Reply:
Fixed in https://github.com/OCFL/spec/pull/654/commits/22f993007a0db2ecc67f1118d29fc5cb7aeb0a7c